dist/bin/scalac

```diff
@@ -75,11 +74,10 @@ eval "\"$JAVACMD\"" \
    ${JAVA_OPTS:-$default_java_opts} \
    "${java_args[@]}" \
    "-classpath \"$jvm_cp_args\"" \
-   -Dscala.usejavacp=true \
-   "$setScriptName" \
-   "$PROG_NAME"  \
-   "${scala_args[@]}" \
-   "${residual_args[@]}" \
-   "${scripting_string-}"
-scala_exit_status=$?
+   "-Dscala.usejavacp=true" \
+   "-Dscala.home=$PROG_HOME" \
```

Comment:
Are these things that the cs runner would be able to pass to `MainGenericRunner`?

Reply:
I don't think so, but this is supported by the Scala script for use in programs, I don't think it should be supported in CS because the directory structure is not equivalent

Reply:
Some scripts will fail if `-Dscala.home=$PROG_HOME` isn't defined.  It might not matter for non-script code, although it can be useful there as well.

Reply:
Idea from @sjrd it could likely be much more useful to provide properties that have the classpath of the scala compiler and scala standard libraries

Reply:
I like this suggestion, it would then be possible to find the first scala3-library*.jar in the classpath, and two directories up would be `scala.home`.